academic-services/knowledge-exploration-service/reference-post-interpret.md

```diff
@@ -0,0 +1,473 @@
+---
+title: POST Interpret
+description: Generates semantic interpretations of a natural language query
+ms.topic: reference
+ms.date: 2020-02-07
+---
+
+# Interpret REST API
+
+The **Interpret** method generates semantic interpretations of a natural language query based on the combination of a language grammar and indexed attributes.
```

Comment:
suggestion:
The Interpret method generates semantic interpretations of a natural language query.

The rest can be covered in the natural language queries page that you have linked below.